pkg/virt-launcher/virtwrap/network/dhcp/dhcp_test.go

```diff
@@ -244,5 +244,18 @@ var _ = Describe("DHCP", func() {
 			}))
 			Expect(options[240]).To(Equal([]byte("private.options.kubevirt.io")))
 		})
+
+		It("Should not set RouterIP or Classless Static Routes options with empty values", func() {
```

Comment:
nit: do you mind splitting this test in two ? One for router IP, another for classless static routes ?

Reply:
Done - used describeTable instead of `It`s, I think it reads better this way